content/1.7/back-office/international/locations/add-new-state.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Could you rename this file "add-new-state.md" by "add-edit-new-state.md" ?

Reply:
Also the title and titles

Reply:
@Saimis777 I reopened this comment because it's not solved

Reply:
Also, you should create a "states" folder: content/1.7/back-office/international/locations/states/add-edit-state.md